JRFMemoryNoodler/JRFMemoryNoodler.m

```diff
@@ -10,30 +10,43 @@
 
 #import "JRFMemoryNoodler.h"
 
+#include <sys/stat.h>
+
 NSString *JRFPreviousBundleVersionKey = @"JRFPreviousBundleVersionKey";
 NSString *JRFAppWasTerminatedKey = @"JRFAppWasTerminatedKey";
 NSString *JRFAppWasInBackgroundKey = @"JRFAppWasInBackgroundKey";
 NSString *JRFAppDidCrashKey = @"JRFAppDidCrashKey";
 NSString *JRFPreviousOSVersionKey = @"JRFPreviousOSVersionKey";
-NSString *JRFAppDidAbortKey = @"JRFAppDidAbortKey";
+static char *intentionalQuitPathname;
 
 @implementation JRFMemoryNoodler
 
 + (void)beginMonitoringMemoryEventsWithHandler:(JRFOutOfMemoryEventHandler)handler
                                  crashDetector:(JRFCrashDetector)crashDetector {
-    
+
     [[self sharedInstance] beginApplicationMonitoring];
     signal(SIGABRT, JRFIntentionalQuitHandler);
     signal(SIGQUIT, JRFIntentionalQuitHandler);
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
-    
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
```

Comment:
IIRC the Application Support directory isn't guaranteed to exist. Will `creat` create it as an intermediate directory if it doesn't? (Perhaps overkill, but it wouldn't hurt to have a unit test to verify that behavior).